### DISCUSSION TITLE
Get results from a Stored Procedure in Prisma

### DISCUSSION THREAD

**Discussion by ebaer-payix** [prisma/prisma#7690] [Q&A]
Hi,

I already posted this question in the Slack General Channel, so please forgive me if someone is already reviewing it there:

Not sure if this is the correct location to ask this question, but:  How do I Call and get the return values from a Stored Procedure using Prisma.

Here is the Stored Procedure Syntax:
CALL GetSequenceBlock('ach', 0, @start, @end), or it can be CALL GetSequenceBlock('ach', 0), if need be...

From what I can find in the documentation, any Prisma call has to start with a SELECT, but that doesn't seem to be allowed by MySQL.
ie: SELECT @start, @end FROM ( CALL GetSequenceBlock('ach', 0, @start, @end) );

Any assistance would be greatly appreciated.

Thanks

eb

**Comment by janpio** [comment]
Related discussion on Slack: https://prisma.slack.com/archives/CA491RJH0/p1623774929172200

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
